ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1194,6 +1194,34 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
             break;
 #endif
 
+        case MAV_CMD_DO_WINCH:
+            // param1 : winch number (ignored)
+            // param2 : action (0=release, 1=length control, 2=rate control). See WINCH_ACTIONS enum.
+            if(!copter.g2.winch.enabled()) {
+                result = MAV_RESULT_FAILED;
+            } else {
+                result = MAV_RESULT_ACCEPTED;
```

Comment:
This can be hit when the winch is enabled, but no backend is selected. This should only be accepted if a backend was actually created.

Reply:
Txs.  enabled() now checks if the backend is created and _TYPE parameter is non-zero.